LeaveManagement/LeaveManagement/src/main/java/com/example/LeaveManagement/service/JwtService.java

```diff
@@ -70,19 +70,38 @@ public UserDetails loadUserByUsername(String username) throws UsernameNotFoundEx
     }
     private Set<SimpleGrantedAuthority> getAuthority(User user) {
         Set<SimpleGrantedAuthority> authorities = new HashSet<>();
-        user.getRole().forEach(role -> {
-            authorities.add(new SimpleGrantedAuthority("ROLE_" + role.getRoleName()));
-        });
+        user.getRole().forEach(role ->
+                authorities.add(new SimpleGrantedAuthority("ROLE_" + role.getRoleName()))
+        );
         return authorities;
     }
 
-    private void authenticate(String userName, String userPassword) throws Exception {
+    private void authenticate(String userName, String userPassword) throws UserDisabledException, InvalidCredentialsException {
         try {
+            // Uncovered code
             authenticationManager.authenticate(new UsernamePasswordAuthenticationToken(userName, userPassword));
         } catch (DisabledException e) {
-            throw new Exception("USER_DISABLED", e);
+            throw new UserDisabledException("USER_DISABLED", e);
         } catch (BadCredentialsException e) {
-            throw new Exception("INVALID_CREDENTIALS", e);
+            throw new InvalidCredentialsException("INVALID_CREDENTIALS", e);
+        }
+    }
+
```

Comment:
Remove empty lines

Reply:
Addressing the PR comments provided above has been resolved.